### DISCUSSION TITLE
Image | Radxa CM3

### DISCUSSION THREAD

**Discussion by agent-shredder** [MichaIng/DietPi#6008] [Image Request]
Formal device information

 * Device name: **Radxa CM3 (IO)**
 
 * Processor
Rockchip RK3566, Quad core Cortex-A55 (ARM v8) 64-bit SoC @ 2.0GHz
(same as in Pine64 SOQuartz)

* RAM
1GB, 2GB, 4GB or 8GB LPDDR4

* Storage
8GB, 16GB, 32GB, 64GB, 128GB (depending on variant) high performance eMMC
 
 * Official product URL
 https://wiki.radxa.com/Rock3/CM 
 https://wiki.radxa.com/Rock3/CM3/IO

 * GitHub resource URL
 https://github.com/radxa-build/radxa-cm3-io/

 * Image download URL
 https://github.com/radxa-build/radxa-cm3-io/releases/latest

Is the SBC officially supported by the Debian installer?

* not sure

If not, is a reliable 3rd party Debian image available for this SBC?

* [yes](https://wiki.radxa.com/Rock3/Debian)

If not, are there install instructions for Debian available?

* [yes](https://wiki.radxa.com/Rock3/Debian)

**Comment by MichaIng** [comment]
Probably our ROCK 3A image boots on CM3 as well?

**Reply by agent-shredder** [reply]
Nope it is not working.
ROCK 3A uses Rockchip RK3568 and CM3 uses RK3566

**Reply by groninge01** [reply]
@MichaIng There is an official radxa debian bullseye image for the cm3. Does the conversion method work on that or does it need to be bookworm?
